### DISCUSSION TITLE
When I search for "superdispatchkeyevent" at "definition", some files cannot be searched.

### DISCUSSION THREAD

**Discussion by chenchuanliang** [oracle/opengrok#4350] [General]
**Describe the bug**
version：opengrok 1.7.24  but I think maybe all the version have this issue.
project：Android7/Android8/Android9/Android10/Android11
When I search for "superdispatchkeyevent" at "definition", some files cannot be searched.
I don't know if there are other files that haven't been searched or have other methods  that haven't been searched.

**To Reproduce**
 Search for "superDispatchKeyEvent" at "Definition",You can see the results：MockWindow.java/InstrumentationTest.java/WindowTest.java/PhoneWindow.java/Window.java,but not have DecorView.java.
Actually , The definition of this method exists in the decorview.java file.
Search for "DecorView.java" at "File Path",Enter file search "superDispatchKeyEvent" by "ctrl+F",you can found this method definition in the file.
If use "Full Search",it can search.

**Expected behavior**
Can search "superDispatchKeyEvent" in "DecorView.java" file.

**Screenshots**

![1](https://user-images.githubusercontent.com/20430797/148715567-ca036a0a-8199-46cf-b273-0b17ce3307a9.png)
![2](https://user-images.githubusercontent.com/20430797/148715570-873db34b-632f-478b-bb94-e83280018b3e.png)

Thank you.

**Comment by vladak** [comment]
I saw some definitions (in unrelated code) missing myself lately, this should be investigated.
